### PR TITLE
fix: Bug- components are rendered over nav bar

### PR DESCRIPTION
**Pull request by chohyeonkim**
## :: Type of change

- [x] Bug fix
- [ ] New feature
- [ ] Refactoring
- [ ] Edit convention

<br />

## :: Description
image

<br />

### CODE REVIEW

File: client/src/components/Header/Header.js [hunk 1]
```diff
@@ -18,4 +18,5 @@ const FixedNav = styled.div`
   margin: 0 auto;
   font-family: 'AirbnbCereal_W_Bk', 'Montserrat', sans-serif, Courier, monospace;
   border-bottom: 1px solid rgb(230, 230, 230);
+  z-index: 9999;
```
Comment (jieunkxx) [inline client/src/components/Header/Header.js:21]: with this z-index, the nav menu goes under the nav.
please set it below the nav menu's z-index

**Review by jieunkxx — CHANGES_REQUESTED** [review]

File: client/src/components/Header/Header.js (hunk 1 above)
Comment (czhaoca) [reply client/src/components/Header/Header.js:21]: It definitely works, do you think it’s to high for 9999

Comment (jieunkxx) [reply client/src/components/Header/Header.js:21]: Screenshot 2023-06-22 at 2 26 14 PM

Comment (chris718) [reply client/src/components/Header/Header.js:21]: We'll merge this to main. Thank you :)

Comment (chris718) [reply client/src/components/Header/Header.js:21]: We may change that block's z-index to flip it on the top. Or using a hamburg menu style.

Comment (chohyeonkim) [reply client/src/components/Header/Header.js:21]: @chris718 Hi christina while you guys are working on that
Could you please check if nav is working ok on smaller window size?

image

Comment (czhaoca) [reply client/src/components/Header/Header.js:21]: It looks like this.
Screenshot 2023-06-22 at 7 45 03 PM

**Pull request merged**